src/hexEditorProvider.ts

```diff
@@ -40,7 +40,7 @@ export class HexEditorProvider implements vscode.CustomEditorProvider<HexDocumen
     async openCustomDocument(
         uri: vscode.Uri,
         openContext: vscode.CustomDocumentOpenContext,
-        token: vscode.CancellationToken
+		_token: vscode.CancellationToken
```

Comment:
Why is this indented strangely?

Reply:
Oh i think I reverted it which brought bad indentation back somehow

Reply:
Nope, needed to merge main